build/switch.sh

```diff
@@ -26,7 +26,7 @@ mv ~/creation/rhino-config/rhino-config /usr/bin
 # Install rhino-deinst onto system
 mkdir ~/creation/rhino-deinst
 cd ~/creation/rhino-deinst
-wget https://raw.githubusercontent.com/rollingrhinoremix/rhino-deinst/master/rhino-deinst
+wget https://github.com/rollingrhinoremix/rhino-deinst/releases/download/1.0.0/rhino-deinst
```

Comment:
use "latest/download" so `https://github.com/rollingrhinoremix/rhino-deinst/releases/latest/download/rhino-deinst` this downloads the latest tagged release so 1.0.0 right now.

Reply:
I see. Are you able to submit this change or would you prefer if I do it?

Reply:
Okay then I'll do it then